plugins/ufm_consumer_plugin/conf/ufm_consumer_plugin.conf

```diff
@@ -0,0 +1,11 @@
+[common]
+port_number=8989
+ssl_apache_port=7443
+apache_port=8888
+
+
+[logs-config]
```

Comment:
need to make sure that UFM consumer logs (console.log etc...) will be written to /opt/ufm/files/log/plugins/ufm_consumer

Reply:
Defined log dir to mapped volume /log
in gv.cfg
Logging log_dir /log
So all the logs will be in /opt/ufm/files/conf/plugins/ufm_consumer/log ?